seeds/zeroex/optimism/zeroex_api_optimism_fills_sample.csv

```diff
@@ -0,0 +1,7 @@
+tx_hash,evt_index,contract_address,block_time,maker,taker,taker_token_address,taker_token_symbol,maker_token_address,maker_token_symbol,taker_token_amount,taker_token_amount_raw,maker_token_amount,maker_token_amount_raw,type,affiliate_address,swap_flag,matcha_limit_order_flag,volume_usd
+0xa26706cfe970d4982bfe2203bdbb56aa015f9e5fb3b039a41b28104c66a2e4f2,9,0xa3128d9b7cca7d5af29780a56abeec12b05a6740,2022-11-22 7:07,0x0000001d56656c6f64726f6d6500000000000000,0x9da25447f556795da0ceb0c5fb59b13bd7089b9b,0x7f5c764cbc14f9669b88837ca1490cca17c31607,USDC,0x4200000000000000000000000000000000000006,WETH,15.753507,15753507,0.01451057854,14510578541438252,Bridge Fill,0x99b36fdbc582d113af36a21eba06bfeab7b9be12,TRUE,FALSE,15.753507
+0x9ee743dd77dee0687bfda2e285eab389d767c13d99c31d605806e7483c2aa4a4,17,0xa3128d9b7cca7d5af29780a56abeec12b05a6740,2022-11-22 7:04,0x0000001d56656c6f64726f6d6500000000000000,0x6408f01d99a4eb79e9d2a8a6e30132f41e33dafa,0x7f5c764cbc14f9669b88837ca1490cca17c31607,USDC,0x4200000000000000000000000000000000000006,WETH,796.153846,796153846,0.733440944,733440944020012300,Bridge Fill,0x7cba0eb7a94068324583be7771c5ecda25e4c4d1,TRUE,FALSE,796.153846
+0x9ee743dd77dee0687bfda2e285eab389d767c13d99c31d605806e7483c2aa4a4,9,0xa3128d9b7cca7d5af29780a56abeec12b05a6740,2022-11-22 7:04,0x00000012556e6973776170563300000000000000,0x6408f01d99a4eb79e9d2a8a6e30132f41e33dafa,0x7f5c764cbc14f9669b88837ca1490cca17c31607,USDC,0x4200000000000000000000000000000000000006,WETH,9553.846154,9553846154,8.797090945,8797090944535965000,Bridge Fill,0x7cba0eb7a94068324583be7771c5ecda25e4c4d1,TRUE,FALSE,9553.846154
+0x5f6612bc917f08d1e98b0a5d0cde86a8b1ae71f89723823f1576d01511ba7f0e,14,0xa3128d9b7cca7d5af29780a56abeec12b05a6740,2022-11-22 7:01,0x00000012556e6973776170563300000000000000,0xac8777e09acae7816ce3c598ca85293dda3e3b72,0x6fd9d7ad17242c41f7131d257212c54a0e816691,UNI,0x94b008aa00579c1307b0ef2c499ad98a8ce58e58,USDT,36.92307692,36923076923076920000,185.758042,185758042,Bridge Fill,0x1000000000000000000000000000000000000011,TRUE,FALSE,185.590126
+0x5f6612bc917f08d1e98b0a5d0cde86a8b1ae71f89723823f1576d01511ba7f0e,6,0xa3128d9b7cca7d5af29780a56abeec12b05a6740,2022-11-22 7:01,0x00000012556e6973776170563300000000000000,0xac8777e09acae7816ce3c598ca85293dda3e3b72,0x6fd9d7ad17242c41f7131d257212c54a0e816691,UNI,0x94b008aa00579c1307b0ef2c499ad98a8ce58e58,USDT,3.076923077,3076923076923076600,15.457785,15457785,Bridge Fill,0x1000000000000000000000000000000000000011,TRUE,FALSE,15.44381194
+0xd07e3eb04bf916eb254e7a4c146a9c3b7af4b0e9c9ecbce656014cd43d34f6e3,26,0xa3128d9b7cca7d5af29780a56abeec12b05a6740,2022-11-22 7:01,0x0000001d56656c6f64726f6d6500000000000000,0x887a0b9e8c35f531967d993f2c6d3c7426731714,0x4200000000000000000000000000000000000042,OP,0x7f5c764cbc14f9669b88837ca1490cca17c31607,USDC,285.3912795,285391279512566560000,231.128147,231128147,Bridge Fill,0x1000000000000000000000000000000000000011,TRUE,FALSE,231.12814
```

Comment:
Hi, @RantumBits , I am using `tx_hash` and `evt_index` to join the seed file and checking `maker` and `taker`.
```
- check_seed:
          seed_file: ref('zeroex_api_optimism_fills_sample')
          match_columns:
            - tx_hash
            - evt_index
          check_columns:
            - maker
            - taker
```
However I am having some unmatched seeds from [the test warning](https://github.com/duneanalytics/spellbook/actions/runs/3648838360/jobs/6162730181#step:12:23).

Reply:
Seems those `tx_hash`+`evt_index` does not appear in the model result.
https://dune.com/queries/1729186?d=1